lib/iron_bank/metadata.rb

```diff
@@ -12,6 +12,13 @@ def excluded_fields
       fields.fetch(object_name, [])
     end
 
+    # NOTE: For some resources, fields are queryable with some restrictions,
```

Comment:
WDYTA moving this note into `IronBank::Resources::Invoice`?

Reply:
This is an example of `single_resource_query_fields` that's likely to be the most understandable. I'd rather leave it where the method is defined.